kratos/includes/model_part.h

```diff
@@ -401,6 +401,12 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
 
     void AssignNode(NodeType::Pointer pThisNode, IndexType ThisIndex = 0);
 
+    /** Returns if the Node corresponding to it's identifier exists */
+    bool HasNode(IndexType NodeId, IndexType ThisIndex = 0)
+    {
+        return GetMesh(ThisIndex).HasNode(NodeId);
+    }
```

Comment:
```suggestion
    bool HasNode(IndexType NodeId)
    {
        return GetMesh(0).HasNode(NodeId);
    }
```

Just wondering if we should start getting rid of the mesh default argument.

Reply:
No my decision...

Reply:
@KratosMultiphysics/technical-committee